sscanss/core/io/reader.py

```diff
@@ -595,3 +598,136 @@ def read_robot_world_calibration_file(filename):
         raise ValueError('Non-finite value present in calibration data')
 
     return result
+
+
+def read_tomoproc_hdf(filename) -> dict:
```

Comment:
this is question to @StephenNneji I would say and for when he get back, and whether appropriate to use type hints in this PR